source/includes/fact-online-archive-callout.rst

```diff
@@ -0,0 +1,5 @@
+If you are archiving stale data to save on storage costs, consider
```

Comment:
I LGTM'd but as a thought - I wonder if we should have an extract of CTAs so we can sort of organize them all in once place.

`extracts-cta.yaml` or something.